### PR TITLE
Tutorials and examples

### PR DESCRIPTION
**Pull request by zasdfgbnm**

### CODE REVIEW

File: doc/dev/tma.md [hunk 1]
```diff
@@ -435,4 +433,48 @@ or something else?).
 We do have the flexibility to arbitrarily transform and parallelize the non-tile branch,
 and scheduling this is very similar to how we schedule other fusions.
 
+### Code walk-through
+
+Please go to [test_tutorial.cpp](../../tests/cpp/test_tutorial.cpp) and search `TEST_F(Tutorial, BasicTMA)`.
+
 ### Examples
+
+#### Example 2: broadcast kernel with discontiguous input
```
Comment (zasdfgbnm) [inline doc/dev/tma.md:442]: Example 1 will be `TEST_F(Tutorial, PointwiseTMA)` in https://github.com/NVIDIA/Fuser/pull/2148, thanks @rdspring1 for creating it!

**Review by naoyam — APPROVED** [review]
LGTM. Thanks for adding the examples.

File: doc/dev/tma.md [hunk 2]
```diff
@@ -435,4 +433,48 @@ or something else?).
...
+T2[b, I1, I2, I3] = broadcast(T0[I1, I2, I3])
+T3[I0, I1, I2, I3] = T1[I0, I1, I2, I3] + T2[b, I1, I2, I3]
+
+outputs:
+T3[I0, I1, I2, I3] contiguity: T, T, T, T
+```
+
+Schedule:
+
+![Example 1: Broadcast + binary Op](tma/example2.svg)
```
[the remark this answers is not in the document]
Comment (zasdfgbnm) [reply doc/dev/tma.md]: fixed

File: doc/dev/tma.md [hunk 3]
```diff
@@ -435,4 +433,48 @@ or something else?).
...
+math:
+T1[I1, I0] = transpose(T0[I0, I1])
+
+outputs:
+T1[I1, I0] contiguity: T, T
+```
+
+Schedule:
+
+![Example 2: Bank-conflict-free transpose](tma/example3.svg)
```
[the remark this answers is not in the document]
Comment (zasdfgbnm) [reply doc/dev/tma.md]: fixed

**Pull request merged**